Games/LLEGSB/script.js

```diff
@@ -0,0 +1,248 @@
+function question4(){
+  document.getElementById('demo').innerHTML = "Question 4";
+  confirmationText.text = '';
+  quitText.text = '';
+  hintText.text = ''
+  var a = Math.floor(Math.random() * (-20 - 20 + 1)) + 20;
+  var b = Math.floor(Math.random() * (10 - 2 + 1)) + 2;
+  var c = Math.floor(Math.random() * (7 - 2 + 1)) + 2;
+  var d = Math.floor(Math.random() * (-20 - 20 + 1)) + 20;
+  var e = Math.floor(Math.random() * (10 - 2 + 1)) + 2;
+  questionText.text = `Combine all like terms in the given expression: ${a}x + ${b}y + ${d}x - ${c}z + ${e}y` 
+  correctAnswer = `${a+d}x+${b+e}y-${c}z`;
+}
+
+// question 5 function
+// slope of a line
+function question5(){
+  document.getElementById('demo').innerHTML = "Question 5";
+  confirmationText.text = '';
+  quitText.text = '';
+  hintText.text = ''
+  var y1 = Math.floor(Math.random() * (200 - 1 + 1)) + 1;
+  var y2 = Math.floor(Math.random() * (200 - 1 + 1)) + 1;
+  var x1 = Math.floor(Math.random() * (200 - 1 + 1)) + 1;
+  var x2 = Math.floor(Math.random() * (200 - 1 + 1)) + 1;
+  questionText.text = `What is the slope of a line that contain the points (${x1}, ${y1}) and (${x2}, ${y2}).`;
+  var slopeEquation = ((y2-y1)/(x2-x1));
+  slopeRounded = slopeEquation.toFixed(2);
+  correctAnswer = `${slopeRounded}`;
+}
+//question 6
+// length of a line
+function question6(){
+  document.getElementById('demo').innerHTML = "Question 6";
+  confirmationText.text = '';
+  quitText.text = '';
+  hintText.text = ''
+  var x1 = Math.floor(Math.random() * (300 - 100 + 1)) + 100;
+  var y1 = Math.floor(Math.random() * (300 - 100 + 1)) + 100;
+  var x2 = Math.floor(Math.random() * (300 - 100 + 1)) + 100;
+  var y2 = Math.floor(Math.random() * (300 - 100 + 1)) + 100;
+  questionText.text = `What is the length of a line segment(to 2 decimal places) with the endpoints (${x1}, ${y1}) and (${x2}, ${y2})`  
+  var x = (x2-x1); 
+  var y = (y2-y2);
+  var xSquare = (x**2);
+  var ySquare = (y**2);
+  var add = (xSquare+ySquare);
+  var squareRoot = (add**0.5);
+  ansRounded = squareRoot.toFixed(2);
+  correctAnswer = ansRounded;
+  
+}
+// question 7 function
+// introduces students to simple grade 10 algebra
+// introduction to FOIL method
+function question7(){
+  document.getElementById('demo').innerHTML = "Question 7";
+  confirmationText.text = '';
+  quitText.text = '';
+  hintText.text = '';
+  var rndmNum1 = Math.floor(Math.random() * (10 - 2 + 1)) + 10;
+  var rndmNum2 = Math.floor(Math.random() * (10 - 2 + 1)) + 10;
+  var expression = `(x+${rndmNum1})(x+${rndmNum2})`;
+  questionText.text = `Expand the expression ${expression}`;
+  var ans1 = `x^2`;
+  var ans2 = `${rndmNum1+rndmNum2}x`;
+  var ans3 = `${rndmNum1*rndmNum2}`;
+  var finalAns = `${ans1}+${ans2}+${ans3}`;
+  correctAnswer = finalAns;
+}
+
+// a function to check the answer for question 
+
+function checkAnswer(answer){
+  hintText.text = '';
+    if (answer == correctAnswer) {
+      (confirmationText.text = 'Correct!')
```

Comment:
You don't need the parens around the statement. 
Similarly below.
```suggestion
      confirmationText.text = 'Correct!';
```